Orm/Xtensive.Orm.SqlServer/Sql.Drivers.SqlServer/v09/Compiler.cs

```diff
@@ -379,12 +379,12 @@ public override void Visit(SqlCreateIndex node, IndexColumn item)
 
     #region Static helpers
 
-    private static SqlCast CastToLong(SqlExpression arg)
+    protected static SqlCast CastToLong(SqlExpression arg)
```

Comment:
Why the access modifier has changed to `protected` here?

Reply:
This and the method bellow left protected accidentally after experimenting with original formula. There is nothing criminal since these methods are accessible within SqlServer providers but I revert them back to private